### PR TITLE
Feature/image block

### PR DESCRIPTION
**Pull request by amomo25**
Avant :

Avant

Après :

Après

### CODE REVIEW

**Review by lucasrlt — APPROVED** [review]
Follows the same structure as other blocks

File: app/src/main/java/com/pindex/main/utils/Constants.kt [hunk 1]
```diff
@@ -50,6 +50,10 @@ object Constants {
          */
         const val TEXT_BLOCK_TEXT: String = "text"
 
+        /**
+         * The image block text argument.
+         */
+        const val IMAGE_BLOCK_TITLE: String = "imagePath"
```
[the remark this answers is not in the document]
Comment (amomo25) [reply app/src/main/java/com/pindex/main/utils/Constants.kt:56]: You're right, it would make it more clear, i'm resolving that right away.

**Review by lucasrlt — APPROVED** [review]
Looks fine to me after Quentin's review

**Review by ToufiPF — APPROVED** [review]
Looks good to me too!

**Pull request merged**